cmake-proxies/cmake-modules/AudacityFunctions.cmake

```diff
@@ -509,6 +509,22 @@ macro( audacity_library NAME SOURCES IMPORT_TARGETS
    set( AUDACITY_LIBRARIES "${AUDACITY_LIBRARIES}" PARENT_SCOPE )
 endmacro()
 
+# A special macro for header only libraries
+
+macro( audacity_header_only_library NAME SOURCES IMPORT_TARGETS
+   ADDITIONAL_DEFINES )
+   # ditto comment in the previous macro
+   add_library( ${NAME} INTERFACE )
+
+   target_include_directories ( ${TARGET} INTERFACE ${CMAKE_CURRENT_SOURCE_DIR} )
```

Comment:
Where is TARGET defined?  The function argument is NAME

Reply:
Now I see TARGET was defined in the context of the sole use of this new CMake function, but I think you should not rely on that, use NAME